models/etcd/dhcp.go

```diff
@@ -76,6 +76,10 @@ func (l *leaseUsage) UnmarshalJSON(data []byte) error {
 		return err
 	}
 
+	if l.hwMap == nil {
+		l.hwMap = make(map[string]leaseInfo)
+	}
+
```

Comment:
In some cases, hwmap will be nil, so we initialize it here.

Reply:
Why do you want to initialize it _here_?
I think it can be initialized where a new key is stored in this map.

Reply:
I think it is better to allocate the map when initializing the "leaseUsage" structure.
In this case, in effect,  this unmarshaller initialize the structure, we're allocating the map here.